### PR TITLE
[one-cmds] Fix one-import-onnx

### PR DESCRIPTION
**Pull request by seanshpark**
This will fix one-import-onnx as env. var can be None.

### CODE REVIEW

File: compiler/one-cmds/one-import-onnx [hunk 1]
```diff
@@ -248,7 +248,7 @@ def _check_ext(args):
     if oneutils.is_valid_attr(args, 'disable_ext'):
         return None
     env_disable_ext = os.getenv('ONE_IMPORT_ONNX_EXT_DISABLE')
-    if env_disable_ext == '1' or env_disable_ext.upper() == 'Y':
+    if env_disable_ext == '1' or env_disable_ext == 'Y':
```
Comment (seanshpark) [inline compiler/one-cmds/one-import-onnx:251]: we can check `env_disable_ext` is `None` or not but I want to go simple check with `1` or `Y`

**Comment by seanshpark** [comment]
to fix https://github.com/Samsung/ONE/issues/13138#issuecomment-2179484380

**Review by Seunghui98 — APPROVED** [review]
LGTM 👍

**Comment by seanshpark** [comment]
CC @shs-park

**Review by shs-park — COMMENTED** [review]
LGTM!
=)

**Pull request merged**